### ISSUE TITLE
Newline when importing from Matroska

### ISSUE THREAD

**Issue by mkver** [SubtitleEdit/subtitleedit#2838]
Hello,

it seems that when ffmpeg converts an srt file (not S_TEXT/UTF8 that is already in Matroska) to an S_TEXT/UTF8 track inside Matroska, it uses only /n. And when SE imports such a track from such a file, it doesn't convert the \n to \r\n as it should do (at least on Windows) so that the files appear as if they were only on one line.
[Here](https://github.com/SubtitleEdit/subtitleedit/files/1821423/test.zip) is a sample.

Thanks again
mkver

**Comment by niksedk** [comment]
thx for the info and sample :)

Beta updated: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.6/SubtitleEditBeta.zip

**Comment by niksedk** [comment]
Comment in typo fixed here: c58dfb150a4775d06e330aa5a3657c74b9703e02

**Comment by mkver** [comment]
I just tested the new beta and it works on the sample I provided. But then I had a look at the code and thought that it might make \r\r\n out of \r\n; it doesn't, because the SplitToLines function treats \r\r\n as a single linebreak (against the spec). But it seems that the SplitToLines function can already handle the whole linebreak stuff on its own and there is no need for the `Replace("\\N", Environment.NewLine)` at all. (Besides the obvious performance improvement this would make importation of subtitles from Matroska work even if you someday decide that \r\r\n is not a single linebreak. I want to add that I don't care about whether \r\r\n is a single linebreak or not.)
